homeassistant/components/comelit/climate.py

```diff
@@ -0,0 +1,200 @@
+"""Support for climates."""
+from __future__ import annotations
+
+import asyncio
+from enum import StrEnum
+from typing import Any
+
+from aiocomelit import ComelitSerialBridgeObject
+from aiocomelit.const import CLIMATE, SLEEP_BETWEEN_CALLS
+
+from homeassistant.components.climate import (
+    ClimateEntity,
+    ClimateEntityFeature,
+    HVACAction,
+    HVACMode,
+    UnitOfTemperature,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import ATTR_TEMPERATURE, PRECISION_TENTHS
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import _LOGGER, DOMAIN
+from .coordinator import ComelitSerialBridge
+
+
+class ClimaAction(StrEnum):
+    """Serial Bridge clima actions."""
+
+    OFF = "off"
+    ON = "on"
+    MANUAL = "man"
+    SET = "set"
+    AUTO = "auto"
+
+
+API_STATUS: dict[str, dict[str, Any]] = {
+    "O": {
+        "action": "off",
+        "hvac_mode": HVACMode.OFF,
+        "hvac_action": HVACAction.OFF,
+    },
+    "L": {
+        "action": "lower",
+        "hvac_mode": HVACMode.COOL,
+        "hvac_action": HVACAction.COOLING,
+    },
+    "U": {
```

Comment:
I see different values: O, U, L, A. For O you already have a constant, would it maybe be nice to have the rest as constant/enum as well?

Reply:
Ok will do later this evening once back home

Reply:
Done